TotalCrossSDK/src/main/java/tc/tools/AnonymousUserData.java

```diff
@@ -86,72 +91,192 @@ private static JSONObject readJsonObject(Stream stream) throws IOException {
         }
     }
 
+    public static HttpURLConnection getConn(String url) throws IOException {
+        HttpURLConnection conn = (HttpURLConnection) new URL(url).openConnection();
+        conn.setRequestMethod("GET");
+        conn.setRequestProperty("Accept", "application/json");
+        conn.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
+        conn.setDoInput(true);
+        conn.setDoOutput(true);
+        conn.setConnectTimeout(60_000);
+        conn.setReadTimeout(60_000);
+        return conn;
+    }
+
+    public static String getReponseBody(HttpURLConnection conn) throws IOException {
```

Comment:
I think things are not well encapsulated here, should probably move these to its own class.

Reply:
its own class or method? I mean, the only problem I see, it's because it's public. To the matter of reusing such an ammount of code, I think it's ok!